src/EnumMap.php

```diff
@@ -54,6 +54,22 @@ public function __construct(string $enumeration, iterable $map = null)
         }
     }
 
+    /**
+     * Add virtual private property "__pairs" with a list of key-value-pairs
+     * to the result of var_dump.
+     *
+     * This helps debugging as internally the map is using the ordinal number.
+     *
+     * @return array<string, mixed>
+     */
+    public function __debugInfo(): array {
+        $dbg = (array)$this;
+        $dbg["\0" . self::class . "\0__pairs"] = array_map(function ($k, $v) {
```

Comment:
Shouldn't this be `static::class` instead because the class is not final?

Reply:
No, because this is debug information only and it should be make the impression to access this.
Adding it as private property of the base class shows it's impossible to access (at least without reflection/closure).

Reply:
Ah, I understand, thx for the clarification :)